scripts/variantstore/beta_docs/gvs-exome-workspace-description.md

```diff
@@ -89,8 +89,11 @@ Below are several examples of the time and cost of running the workflow. General
 
```

Comment:
It wouldn't let me comment on parts that weren't changed, but line 87 is where it says TBD ^